addons/es/verifactu/tax.go

```diff
@@ -13,15 +13,13 @@ var taxCategoryOpClassMap = tax.Extensions{
 	tax.RateReduced:      "S1",
 	tax.RateSuperReduced: "S1",
 	tax.RateZero:         "S1",
-	tax.RateExempt:       "N1", // General exemption, no tax applies
-	tax.RateExempt.With(tax.TagReverseCharge): "S2",
-	tax.RateExempt.With(tax.TagExport):        "N2",
 }
 
-func normalizeTaxCombo(tc *tax.Combo) {
+func normalizeTaxCombo(tc *tax.Combo, isEuropeanCustomer bool) {
```

Comment:
Using boolean flags like this is generally not recommend mainly because its hard to scale.